crates/node-e2e-tests/tests/it/eth.rs

```diff
@@ -116,6 +116,52 @@ async fn can_run_eth_node() -> eyre::Result<()> {
     Ok(())
 }
 
+#[tokio::test]
+async fn can_run_eth_node_with_auth_engine_api_over_ipc() -> eyre::Result<()> {
+    let tasks = TaskManager::current();
+    let test_suite = TestSuite::new();
+
+    // Node setup
+    let node_config = NodeConfig::test()
+        .with_chain(test_suite.chain_spec().clone())
+        .with_rpc(RpcServerArgs::default().with_http());
```

Comment:
this is still weird to me, 
it looks like auth ipc is now enabled by default:

```
2024-04-09T17:23:06.720723Z  INFO RPC auth server started url=127.0.0.1:8551 ipc_endpoint=/tmp/reth_engine_api.ipc
```

which should not be the case

Reply:
yeap, I fixed it and make a second test to check the behaviour to see how it works